app/CMakeLists.txt

```diff
@@ -115,6 +115,17 @@ if(OPENXR)
                 ${CMAKE_SOURCE_DIR}/../third_party/hvr/${ANDROID_ABI}/libxr_loader.so
                 ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/libxr_loader.so
                 )
+    elseif (PICOOXR)
+        include_directories(
+            ${CMAKE_SOURCE_DIR}/../third_party/OpenXR-SDK/include
+            ${CMAKE_SOURCE_DIR}/../third_party/picooxr/OpenXR/Include
```

Comment:
We don't need to strictly adhere to the packaging of the SDK. In the third party repo we usually have a folder per platform and then just the minimum required files for the build. This means that it's enough to have
```
picooxr/Include
picooxr/Libs
```

Reply:
I'd prefer to keep the original folder structure, as that makes it easier for people without access to your private repository to build the project. This is independent of not adding unnecessary files to your private repository: you can just omit them when setting up your repository.

That said, in this specific case, I could live with dropping the `OpenXR` folder if it is a big deal to you, as it seems you keep the structure identical (including the uncommon capitalization made by Pico).